### ISSUE TITLE
Error: Variables must be length 1 or 1166. Problem variables: 'text'

### ISSUE THREAD

**Issue by davemcg** [sahirbhatnagar/manhattanly#7]
Flat out isn't working. Even with the included HapMap data. Error is in the subject.

> sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.11.6 (El Capitan)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] devtools_1.12.0   manhattanly_0.1.0 qqman_0.1.2       dplyr_0.5.0       purrr_0.2.2       readr_1.0.0       tidyr_0.6.0      
 [8] tibble_1.2        tidyverse_1.0.0   data.table_1.9.6 

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.7       formatR_1.4       git2r_0.15.0      plyr_1.8.4        highr_0.6         base64enc_0.1-3   tools_3.3.1      
 [8] digest_0.6.10     jsonlite_1.1      memoise_1.0.0     evaluate_0.10     gtable_0.2.0      viridisLite_0.1.3 DBI_0.5-1        
[15] curl_2.2          withr_1.0.2       httr_1.2.1        stringr_1.1.0     knitr_1.15        htmlwidgets_0.8   grid_3.3.1       
[22] R6_2.2.0          plotly_4.5.6      magrittr_1.5      scales_0.4.1      htmltools_0.3.5   assertthat_0.1    colorspace_1.2-7 
[29] stringi_1.1.2     lazyeval_0.2.0    munsell_0.4.3     chron_2.3-47

**Comment by sahirbhatnagar** [comment]
Thanks for reporting this. Can you try using the development version on github?

``` R
install.packages("devtools")
devtools::install_github("sahirbhatnagar/manhattanly")
```

**Comment by davemcg** [comment]
I am

**Comment by sahirbhatnagar** [comment]
Can you provide the code that produced that error?

**Comment by davemcg** [comment]
```
> manhattanly(HapMap)
Error: Variables must be length 1 or 1166.
Problem variables: 'text'
```

**Comment by sahirbhatnagar** [comment]
I've located the bug. Im working on it now. For the time being, you can get around this by specifying an annotation column:

``` R
manhattanly(HapMap, snp = "SNP")
```

**Comment by davemcg** [comment]
Great, thanks

**Comment by sahirbhatnagar** [comment]
Fixed #8
